lib/account/pages/about_page.dart

```diff
@@ -9,15 +12,101 @@ class AboutPage extends StatefulWidget {
 }
 
 class _AboutPageState extends State<AboutPage> {
+  PackageInfo? _packageInfo;
+
+  @override
+  void initState() {
+    super.initState();
+    PackageInfo.fromPlatform().then(
+      (value) => setState(
+        () => _packageInfo = value,
+      ),
+    );
+  }
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
         title: Text(S.of(context).pageAboutTitle),
       ),
       body: Center(
-        child: SingleChildScrollView(
-          child: Text(S.of(context).appName),
+        child: CustomScrollView(
+          physics: const ClampingScrollPhysics(),
+          slivers: [
+            SliverFillRemaining(
+              hasScrollBody: false,
+              child: Padding(
+                padding: const EdgeInsets.all(10),
+                child: Column(children: [
+                  Expanded(child: Container()),
+                  Text(
+                    S.of(context).appName,
+                    style: Theme.of(context).primaryTextTheme.displaySmall,
```

Comment:
Is this supposed to look this weird? 
<img width="332" alt="image" src="https://user-images.githubusercontent.com/63241108/211675377-83c87601-22ef-479a-a756-4296a8dd81a2.png">

This seems to be the wrong style.

Reply:
Good point, I only tested this with dark mode. Fixed

Reply:
Nice, thanks! I stillthink this is maybe too big? In english, it already flows into the second line on the device I'm testing on

Reply:
Better?